backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -28,6 +28,7 @@ class Arguments:
     ok = graphene.Boolean()
     story = graphene.Field(lambda: StoryResponseDTO)
 
+    @require_authorization_by_role_gql({"User", "Admin"})
```

Comment:
Not sure if `CreateStory` mutation can be admin only.

Reply:
lets do admin only